packages/components/src/tabs/stories/index.story.tsx

```diff
@@ -70,6 +70,15 @@ const Template: StoryFn< typeof Tabs > = ( props ) => {
 
 export const Default = Template.bind( {} );
 
+export const RTL = () => {
+	return (
+		<div dir="rtl" style={ { padding: '1rem' } }>
+			{ /* @ts-expect-error - This is a temporary hack until we migrate to CSF 3 and my sanity is restored. */ }
+			<Template />
+		</div>
+	);
+};
+
```

Comment:
Since Storybook currently has an option to switch language direction, is this unnecessary?

https://gyazo.com/1c2ca76337aeba4501116d172f858ab5

Reply:
TIL! Removing the story.